staging/public-swarm.yaml

```diff
@@ -113,11 +113,11 @@ grafana:
     datasources.yaml:
       apiVersion: 1
       datasources:
-      - name: metrics
```

Comment:
@skylenet you sure this is right?

Reply:
@nonsense yeah it's fine. It's just my editor that likes to make YAML files prettier now, once I save it.